README.md

```diff
@@ -38,11 +38,14 @@ After the installation you can configure the plugin by adding the following to y
 ```lua
 require('adr').setup({
     template_dir = "~/adr-templates",
+    doc_dir = "/docs/",
```

Comment:
Can we change this to `"docs/"`, such that it is configurable by a relative path?

With the current configuration it starts with a `/`, which could cause a confusion because one would assume that this is an absolute path.